docs/usage/filters/interactive-output-filtering.md

```diff
@@ -0,0 +1,32 @@
+---
+id: interactive-output-filtering
+title: "Interactive output filtering"
+sidebar_position: 3
+---
+
+Sometimes you can get long response from a certain command and you may want to filter that to see a subset of the actual result.
+while you can achieve that with `--filter` param at the end of your command, there is another option to handle that with interactive
+output filtering. Interactivity is achieved via an input action text box where you can add your filter criteria as text and press `Enter`.
```

Comment:
Please sync with @ezodude about the link to the section.

```suggestion
For each communication platform, you can use the `--filter` flag at the end of your command. To learn more, see the [Flags](./index.md#flags) section.

If you use the the [Slack integration](../installation/socketslack/index.md) that supports interactive messages, there is another option to handle that: interactive output filtering. Interactivity is achieved via an input action text box where you can add your filter criteria as text and press the **Enter** button.
```

Reply:
Done

Reply:
Will coordinate.